packaging/fedora/azure-nvme-utils.spec.tmpl

```diff
@@ -0,0 +1,34 @@
+Name:           azure-nvme-utils
+Version:        __VERSION__
+Release:        __RELEASE__%{?dist}
+Summary:        Utility and udev rules to help identify Azure NVMe devices
+
+License:        MIT
+URL:            https://github.com/Azure/%{name}
+Source0:        azure-nvme-utils_dev.tgz
```

Comment:
I think this is fine, but if you wanted to get fancy you could use https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_using_forges_hosted_revision_control. Presumably you'll be tagging releases for the Fedora package so we'd need to adjust it regardless, so it's probably not worth it.

Reply:
yeah this will be done differently for proper Fedora packaging, we'll do whatever makes the most sense there when we get there.  I have a couple minor fixes I want to address here first then will start the upstream packaging process